quizsolution.ipynb

```diff
@@ -0,0 +1,161 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "44aa06e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "8a45b728",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "f=open('contact.csv','w',newline='')"
```

Comment:
write in a other way to to open a file ?

Reply:
use dictreader to open file...